### ISSUE TITLE
Hidden element has focusable content

### ISSUE THREAD

**Issue by GaryRidgway** [uiowa/uiowa#9301]
On the facilities [Building Coordinator List](https://facilities.uiowa.edu/building-coordinators-network/building-coordinator-list), the hidden width measurer implemented to not have the table jump around when the details are opened is throwing errors.

It seems as though `aria-hidden='true'` is not enough for screen readers, and the content after it is still navigable.

https://my2.siteimprove.com/Inspector/1005109/A11Y/Page?pageId=65055813775&impmd=6F4B1BAAA780A4D7A3CA00D93433A366&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r17/failed/R1BLaHRXN3A=;MWQxNDI3NDE=

# Proposed solution

add the `visibility: hidden;` rule to the `.width-measurer ` scss rules. This removed it from being an issue when I ran an SI extension scan.
